inc/apirest.class.php

```diff
@@ -446,6 +446,11 @@ public function parseIncomingParams($is_inline_doc = false) {
       if (function_exists('getallheaders')) {
          //apache specific
          $headers = getallheaders();
+         if (false !== $headers) {
+            foreach ($headers as $key => $value) {
+               $headers[ucwords(strtolower($key), '-')] = $value;
```

Comment:
not mandatory, but before adding the new one, i think removing old key could avoid strange behavior, aka:
```unset($headers[$key]);```

It will avoid to have some "duplicates" keys like 
```
[
   'authorization': stuff,
   'Authorization': stuff
]
```